src/test/java/io/stargate/sgv2/jsonapi/api/model/command/builders/SortClauseBuilderTest.java

```diff
@@ -340,18 +340,6 @@ public void mustBeObject() {
       assertThat(throwable).isInstanceOf(JsonApiException.class);
     }
 
-    @Test
-    public void mustNotContainBlankString() {
```

Comment:
Rules do not actually prevent blank sort value (as per Field validation rules), so test was invalid (but implementation not matching either hence passed)

Reply:
I think CollectionSortClauseBuilder.validateSortExpressionPath() is only testing empty

Reply:
Right, blank is allowed so specifically only checking for empty as not allowed (and added blank as allowed).